### PR TITLE
FWF-4078, FWF-3981: [BugFix] Added loader ui , removed formio component panel

### PR DESCRIPTION
**Pull request by Bonymol-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-4078, https://aottech.atlassian.net/browse/FWF-3981
Issue Type: BUG

# Changes
* Added loader ui in input box - updated in formbuilder
* Removed unwanted formio component panel

### CODE REVIEW

File: forms-flow-components/src/components/CustomComponents/FormBuilderModal.tsx [hunk 1]
```diff
@@ -97,9 +99,13 @@ export const FormBuilderModal: React.FC<BuildFormModalProps> = React.memo(
...
-       if((!values.title || values.title !== cachedTitle) && !isCancelButton){
-        nameValidationOnBlur({...values,createButtonClicked})
-        setCachedTitle(values.title); //caching this title to avoid calling handling blur
+      if((!values.title || values.title !== cachedTitle) && !isCancelButton){
+        setIsValidating(true); 
+        nameValidationOnBlur({ ...values, createButtonClicked });
+        setCachedTitle(values.title); 
+        setTimeout(() => {
+          setIsValidating(false); 
+        }, 1000);
```
Comment (arun-s-aot) [inline forms-flow-components/src/components/CustomComponents/FormBuilderModal.tsx:108]: handle it from list.js page. 

i

File: forms-flow-components/src/components/CustomComponents/FormBuilderModal.tsx [hunk 2]
```diff
@@ -76,8 +75,11 @@ export const FormBuilderModal: React.FC<BuildFormModalProps> = React.memo(
       display: checked ? "wizard" : "form"
     })
     const [cachedTitle, setCachedTitle] = useState("");
+    const [isLoading, setIsLoading] = useState(false);
+    const [isValidating, setIsValidating] = useState(false);
```
Comment (arun-s-aot) [inline forms-flow-components/src/components/CustomComponents/FormBuilderModal.tsx:79]: handle this from opensource : 

pass this as props : 
isSaveBtnLoading={formSubmitted}
isFormNameValidating = {validationLoading}

Comment (Bonymol-aot) [reply forms-flow-components/src/components/CustomComponents/FormBuilderModal.tsx]: Resolved

File: forms-flow-components/src/components/CustomComponents/FormBuilderModal.tsx (hunk 1 above)
Comment (Bonymol-aot) [reply forms-flow-components/src/components/CustomComponents/FormBuilderModal.tsx]: Resolved

File: forms-flow-components/src/components/CustomComponents/FormBuilderModal.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Bonymol-aot) [reply forms-flow-components/src/components/CustomComponents/FormBuilderModal.tsx]: Resolved

File: forms-flow-components/src/components/CustomComponents/FormBuilderModal.tsx [hunk 3]
```diff
@@ -147,6 +153,7 @@ export const FormBuilderModal: React.FC<BuildFormModalProps> = React.memo(
               value={values.title}
               isInvalid={!!nameError}
               feedback={nameError}
+              isValidating={isValidating}
```
[the remark this answers is not in the document]
Comment (Bonymol-aot) [reply forms-flow-components/src/components/CustomComponents/FormBuilderModal.tsx]: Updated

**Pull request merged**